WriterDesk1/backend/app/loginapi/routes.py

```diff
@@ -262,4 +262,22 @@ def setUsername():
     current_user.username = newUsername
     # update the database
     db.session.commit()
-    return 'Successfully changed username!', 200
\ No newline at end of file
+    return 'Successfully changed username!', 200
+
+from app.ATP.initialiseATP import initialiseATPNoFiles, initialiseATP
+
+@bp.route("/initializeNoFiles", methods=["POST"])
+def initalizeNoFiles():
```

Comment:
Comments nodig of niet?

Reply:
Deze functie moet weg in productie, dus imo niet